assets/airports/ktpa.json

```diff
@@ -0,0 +1,1566 @@
+        "BAYZZ": ["N27d51.73", "W82d37.28"],
+        "BLOND": ["N27d54.86", "W83d09.81"],
+        "BRDGE": ["N27d27.15", "W82d08.06"],
+        "BULZI": ["N30d22.42", "W84d04.57"],
+        "CAMJO": ["N30d30.53", "W82d41.18"],
+        "CARRD": ["N30d57.52", "W82d27.94"],
+        "COVIA": ["N27d56.18", "W84d44.16"],
+        "CROWD": ["N27d40.02", "W81d57.67"],
+        "CTY"  : ["N29d35.94", "W83d02.93"],
+        "DADES": ["N28d21.76", "W82d15.13"],
+        "DADPE": ["N28d00.34", "W82d36.90"],
+        "DARBS": ["N28d12.00", "W82d51.95"],
+        "DEAKK": ["N27d27.16", "W82d08.06"],
+        "DIPAY": ["N28d16.38", "W82d54.69"],
+        "DORMR": ["N27d38.64", "W83d16.43"],
+        "DUNKN": ["N31d58.70", "W81d57.93"],
+        "ECAKE": ["N28d40.21", "W82d35.31"],
+        "EDISN": ["N29d24.24", "W82d20.22"],
+        "ENDED": ["N28d24.59", "W82d47.07"],
+        "FINKI": ["N28d11.89", "W82d34.56"],
+        "FOOXX": ["N28d05.01", "W82d42.15"],
+        "GANDY": ["N27d46.09", "W82d34.45"],
+        "GNV"  : ["N29d41.53", "W82d16.38"],
+        "GRASN": ["N29d07.94", "W82d15.85"],
+        "GUSNE": ["N28d16.79", "W82d36.45"],
+        "GUZDA": ["N27d59.44", "W82d24.92"],
+        "HALLR": ["N27d32.68", "W81d35.93"],
+        "HEVVN": ["N29d49.32", "W83d53.71"],
+        "JAYJA": ["N29d35.21", "W82d21.53"],
+        "JKBAL": ["N28d17.38", "W82d24.13"],
+        "JSTRM": ["N27d41.51", "W82d25.35"],
+        "KLIIN": ["N29d02.31", "W83d08.26"],
+        "KOKOH": ["N28d31.58", "W82d15.26"],
+        "LACEN": ["N28d34.47", "W82d48.68"],
+        "LAL"  : ["N27d59.17", "W82d00.83"],
+        "LBV"  : ["N26d49.69", "W81d23.49"],
+        "LEGGT": ["N29d13.38", "W83d30.64"],
+        "LISAF": ["N28d11.81", "W82d24.62"],
+        "LZARD": ["N28d18.01", "W82d18.71"],
+        "MARVI": ["N28d38.36", "W82d15.39"],
+        "MILEJ": ["N27d44.49", "W82d11.03"],
+        "MURDO": ["N27d19.12", "W82d19.77"],
+        "OCF"  : ["N29d10.65", "W82d13.58"],
+        "OLENE": ["N28d44.39", "W82d15.48"],
+        "OOSAF": ["N28d04.09", "W82d36.64"],
+        "ORL"  : ["N28d32.56", "W81d20.10"],
+        "PAIRS": ["N27d34.15", "W82d28.79"],
+        "PHK"  : ["N26d46.96", "W80d41.49"],
+        "PIE"  : ["N27d54.47", "W82d41.06"],
+        "PSTOL": ["N27d33.48", "W84d09.92"],
+        "ROGAN": ["N27d15.05", "W81d53.57"],
+        "RSW"  : ["N26d31.79", "W81d46.55"],
+        "RUDOF": ["N27d55.20", "W83d43.14"],
+        "SABEE": ["N27d05.27", "W82d11.49"],
+        "SIMMR": ["N27d47.04", "W84d12.26"],
+        "SPRNG": ["N30d00.21", "W82d41.37"],
+        "SRQ"  : ["N27d24.42", "W82d33.82"],
+        "SYKES": ["N27d39.88", "W83d02.99"],
+        "SZW"  : ["N30d33.37", "W84d22.44"],
+        "TABIR": ["N28d25.19", "W83d00.20"],
+        "TAY"  : ["N30d30.28", "W82d33.17"],
+        "TUMPY": ["N28d10.58", "W82d25.78"],
+        "WILON": ["N29d25.07", "W82d41.59"],
+        "WOBKI": ["N27d41.89", "W82d37.61"]
+    },
+    "restricted": [
+
```

Comment:
please remove empty line

Reply:
Forgot to remove it when I realised there were no restricted areas inside the TRACON :man_facepalming: fixed 👍